CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing
+
+A little check list of contributions
+
+ - Create an issue if you want add a bigger feature and lets talk about it
+ - Take a look at the documentation [development.md](docs/development.md)
+ - Take a look at the [pull request template](.github/PULL_REQUEST_TEMPLATE.md)
+ - If you need help don't hesitate to ask questions in pull requests or issues
+ 
```

Comment:
That covers the majority of my questions. I guess the one thing to me that is missing is tackling a previously made issue. If you want someone to help you out on an issue how would I know which ones I could work. Ya know, trying to handle the case of duplication of efforts.

Reply:
Good point, I use issues at the moment mainly as my personal to-do list. 
I added a little section about tickets marked as `help wanted`.